File: uoo6uoow/Week10/달팽이는 올라가고 싶다.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int a = Integer.parseInt(input[0]);
        int b = Integer.parseInt(input[1]);
        int v = Integer.parseInt(input[2]);

        int day = (v - b - 1) / (a - b) + 1;

        System.out.println(day);
    }
}

File: noeyeyh/Week10/달팽이는 올라가고 싶다.py
a, b, v = map(int, input().split())
days=1

if v <= a:
	print(days)
elif (v - b) % (a - b) == 0:
  days = (v - b) // (a - b)
  print(days)
else:
  days = (v - b) // (a - b) + 1
  print(days)